### PR TITLE
Merge office addin taskpane angular

### PR DESCRIPTION
**Pull request by TCourtneyOwen**

### CODE REVIEW

File: test/ui-test.ts [hunk 1]
```diff
@@ -24,8 +25,9 @@ hosts.forEach(function (host) {
             assert.equal(serverResponse["status"], 200);
 
             // Call startDebugging to start dev-server and sideload
-            const devServerCmd = `npm run dev-server -- --config ./test/webpack.config.js`;
-            await startDebugging(manifestPath, AppType.Desktop, toOfficeApp(host), undefined, undefined, devServerCmd);
+            const devServerCmd = `npm run dev-server -- --config ./test/webpack.config.js `;
+            const devServerPort = parseNumber(process.env.npm_package_config_dev_server_port || 3000);
+            await startDebugging(manifestPath, AppType.Desktop, toOfficeApp(host), undefined, undefined, devServerCmd, devServerPort);
```
Comment (akrantz) [inline test/ui-test.ts]: are there changes needed for stopDebugging to not call "npm run unload"?

Comment (akrantz) [reply test/ui-test.ts]: (I can't tell from the diff view...)

Comment (akrantz) [reply test/ui-test.ts]: I found how to expand it. It looks OK.

Comment (TCourtneyOwen) [reply test/ui-test.ts]: I think this is correct - this is just a straight merge of Angular Ts

**Pull request merged**